### PR TITLE
feat(dws): add new resource to associate users with queue

### PR DESCRIPTION
**Pull request by wuzhuanhong**
**What this PR does / why we need it**:

1. Add new resource to associate users with queue.
2. add related document and acceptance test.

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.

```
make testacc TEST=./huaweicloud/services/acceptance/dws TESTARGS='-run TestAccWorkloadQueueUserAssociate_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dws -v -run TestAccWorkloadQueueUserAssociate_basic -timeout 360m -parallel 4
=== RUN   TestAccWorkloadQueueUserAssociate_basic
=== PAUSE TestAccWorkloadQueueUserAssociate_basic
=== CONT  TestAccWorkloadQueueUserAssociate_basic
--- PASS: TestAccWorkloadQueueUserAssociate_basic (108.34s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dws       108.396s
```

* [x] Documentation updated.
* [x] Schema updated.
* [x] CheckDeleted.

  - **a. During query operation (Read Context)**
    aa. Resource not found
    image

    ab. Related resources (parent resources) not found
   The cluster ID does not exist.
    image

     The workload queue name does not exist.
     image

  - **b. During delete/disassociate/unbind operation (Delete Context)**
    ba. Resource not found
    
       image

    bb. Related resources (parent resources) not found
    The cluster ID does not exist.
     image

### CODE REVIEW

File: docs/resources/dws_workload_queue_user_associate.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+subcategory: "GaussDB(DWS)"
+layout: "huaweicloud"
+page_title: "HuaweiCloud: huaweicloud_dws_workload_plan"
```
Comment (ruwenqiang123) [inline docs/resources/dws_workload_queue_user_associate.md]:
```suggestion
page_title: "HuaweiCloud: huaweicloud_dws_workload_queue_user_associate"
```

**Comment by wuzhuanhong** [comment]
make testacc TEST=./huaweicloud/services/acceptance/dws TESTARGS='-run TestAccWorkloadQueueUserAssociate_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dws -v -run TestAccWorkloadQueueUserAssociate_basic -timeout 360m -parallel 4
=== RUN   TestAccWorkloadQueueUserAssociate_basic
=== PAUSE TestAccWorkloadQueueUserAssociate_basic
=== CONT  TestAccWorkloadQueueUserAssociate_basic
--- PASS: TestAccWorkloadQueueUserAssociate_basic (119.16s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dws       119.219s

**Comment by wuzhuanhong** [comment]
make testacc TEST=./huaweicloud/services/acceptance/dws TESTARGS='-run TestAccWorkloadQueueUserAssociate_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dws -v -run TestAccWorkloadQueueUserAssociate_basic -timeout 360m -parallel 4
=== RUN TestAccWorkloadQueueUserAssociate_basic
=== PAUSE TestAccWorkloadQueueUserAssociate_basic
=== CONT TestAccWorkloadQueueUserAssociate_basic
--- PASS: TestAccWorkloadQueueUserAssociate_basic (119.16s)
PASS
ok github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dws 119.219s

File: docs/resources/dws_workload_queue_user_associate.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The resource ID, consists of `cluster_id` and `queue_name`, separated by a slash.
+
+## Import
+
+The resource can be imported using `id`, e.g.
+
+```bash
+$ terraform import huaweicloud_dws_workload_queue_user_associate.test <id>
```
Comment (Lance52259) [inline docs/resources/dws_workload_queue_user_associate.md]:
```suggestion
The resource can be imported using `cluster_id` and `queue_name` (also `id`), separated by a slash, e.g.

```bash
$ terraform import huaweicloud_dws_workload_queue_user_associate.test <cluster_id>/<queue_name>
```

Comment (wuzhuanhong) [reply docs/resources/dws_workload_queue_user_associate.md]: fixed

**Pull request merged**